unfurl/support.py

```diff
@@ -35,9 +35,15 @@
 
 logger = logging.getLogger("unfurl")
 
-Status = IntEnum(
-    "Status", "unknown ok degraded error pending absent", start=0, module=__name__
-)
+
+class Status(IntEnum):
```

Comment:
This is longer version of the same thing but it allows PyCharm to utilize autocomplete.